docs/how-to-backup-your-Hubs-instance.md

```diff
@@ -1,132 +1,76 @@
+     
 
-![Capture of DigitalOcean, Manage, Backups & Snapshots, Shapshots page. Notification briefly appears in upper right corner with green text.](img/backups/image8.png)
+## **How to restore a backup**
 
-## **Restoring your Hubs data from a Snapshot**
+1. Open VS Code. Be sure to be in the community-edition folder. These instructions will, by default, restore the latest or most recent backup. 
 
-1\. To restore, from the Snapshots tab, Droplets tab, select **More** dropdown menu. Select **Restore Droplet**.
+2. Enter   
+     
+   `npm run restore-backup`
 
-![Capture of DigitalOcean, Manage, Backups & Snapshots, Snapshots tab. More menu highlighted in purple. Choices are Rename, Create Droplet, Add to region, Transfer Snapshot, Restore Droplet, and Delete. "Restore Droplet" is highlighted in purple.](img/backups/image9.png)
+   However, you can restore a specific backup by specifying which backup for the process to use. In that case, you’ll add the actual file name like this: `data_backup_<XXX>`
+ 
+   For example: 
 
-Notification box: Are you sure that you want to restore \[your old droplet\] from \[your saved snapshot\]? Note: restoring will replace the current Droplet with an older image.
+   `npm run restore-backup data_backup_1753910374452`
 
-2\. Select **Restore Droplet**.
+   ![Capture of VS Code, Terminal window, beginning restore. Text of deployment apps "coturn" and other apps deleted.](img/backups/image4.png)
 
-![Capture of DigitalOcean, Manage, Backups & Snapshots page. "Snapshots" tab. Restore droplet popup with "Restore droplet" blue button highlighted in purple.](img/backups/image10.png)
+4. With respect to size and internet connection speed, this will take a few minutes. In our testing, it took us between five and 30 minutes, but an instance with a lot of data, or over a slow Internet connection, it could take hours. 
 
-Notification that Restoring is in progress.
+5. Once the script is finished, the terminal will display ‘instance restarted’.   
 
-![Capture of DigitalOcean, Manage, Backups & Snapshots, Snapshots page. Notification of "Restoring" on right side of page highlighted in purple.](img/backups/image11.png)
+   ![Capture of VS Code, Terminal window, text "instance restarted" highlighted.](img/backups/image5.png)
 
-All done! Notification: Droplet Restored from Snapshot.
+Your instance is back up and running. All of your data should be there. Close VS Code.
 
-![Capture of DigitalOcean, Manage, Backups & Snapshots, Snapshots page. Notification of "Restoring" on right side of page highlighted in purple.](img/backups/image12.png)
 
 ## **After Restoring**
 
-It will take a short while for everything to be restored. DO might indicate that the droplet is available and running but it takes a few minutes for Hubs to restart itself.  If you try your Hubs immediately, you will get broken webpages.
+It will take a short while for everything to be restored. It takes a few minutes for Hubs to restart itself.
```

Comment:
Looking at this again now, do we need the "After Restoring" section anymore, given that just above we say that "Your instance is back up and running. All of your data should be there."?